packages/remix-eslint-config/package.json

```diff
@@ -43,17 +43,20 @@
   "devDependencies": {
     "@types/eslint": "^8.4.6",
     "eslint": "^8.23.1",
+    "jest": "^27.5.1",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0",
     "typescript": "^4.7.4"
   },
   "peerDependencies": {
     "eslint": "^8.0.0",
+    "jest": "^27.0.0 || ^28.0.0 || ^29.0.0",
```

Comment:
```suggestion
```

I don't think we should add this, even if marked as optional. You can use Jest's rules without Jest if your test runner share's Jest's API (I use them with projects that use Vitest).

Reply:
I added this because `jest-testing-library` config didn't work without `jest`
Didn't test with `vitest` though.

Since the config has `jest` in it's name, I think we should at least add it to `peerDependencies` imo.
What do you think about adding `vitest` (& making it optional) as well?

Reply:
I still say remove it. Many of our stacks use Vitest and these rules should work well since they share most of the same API.

Reply:
I still say remove it. Many of our stacks use Vitest and these rules should work well since they share most of the same API.